x-pack/plugins/security_solution/public/management/components/endpoint_responder/endpoint_response_actions_console_commands.tsx

```diff
@@ -62,21 +92,58 @@ const COMMENT_ARG_ABOUT = i18n.translate(
   { defaultMessage: 'A comment to go along with the action' }
 );
 
-export const getEndpointResponseActionsConsoleCommands = (
-  endpointAgentId: string
-): CommandDefinition[] => {
+const DISABLED_COMMAND_INFO = i18n.translate(
+  'xpack.securitySolution.endpointConsoleCommands.suspendProcess.disabledCommandInfo',
+  { defaultMessage: 'This endpoint does not support this commmand' }
```

Comment:
This should have a message about updating the Endpoint.

"This version of the Endpoint does not support this command.  Upgrade your Agent in Fleet to use the latest response actions."